src/encoded/schemas/fvp_b6.json

```diff
@@ -0,0 +1,355 @@
+{
+    "title": "NACC Uniform Data Set (UDS) - FOLLOW-UP FORM B6: BEHAVIORAL ASSESSMENT - GERIATRIC DEPRESSION SCALE (GDS[1])",
+    "description": "INSTRUCTIONS: This form is to be completed by the clinician or other trained health professional, based on subject response. For additional clarification and examples, see UDS Coding Guidebook for Follow-up Visit Packet, Form B6. Link to NACC Coding Guidebook Check only one answer per question.",
+    "id": "/profiles/uds_fvp_b6.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "identifyingProperties": [
+        "uuid"
+    ],
+    "required": [
+        "fu_note_b6"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/header"
+        },
+        {
+            "$ref": "mixins.json#/examinier_initials"
+        },
+        {
+            "$ref": "mixins.json#/form_status"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        }
+    ],
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "fu_nogds": {
+            "title": "Check this box and enter \"88\" below for the Total GDS Score if and only if the subject: 1.) does not attempt the GDS, or 2.) answers fewer than 12 questions.",
+            "type": "string",
+            "enum": [
+                "1"
```

Comment:
Do we need enum here?

Reply:
Will change type to  boolean, keep same type with ivp_b6.

Reply:
![image](https://user-images.githubusercontent.com/38848214/100903421-8025ee00-348b-11eb-8fd1-a04dbaa9f4c5.png)